styles.css

```diff
@@ -136,3 +140,107 @@ body {
 .main__button:hover {
   opacity: .5;
 }
+
+@media (max-width:667px) {
+  #checkbox {
+    display: none;
+  }
+
+/* .container, .logo__circle {
+  margin: 15px 0;
+} */
+
+  .rectangle1 {
+    position: absolute;
+    width: 25px;
+    height: 3px;
+    left: 326px;
+    top: 32px;
+    background: #0056EC;
```

Comment:
Codacy found an issue: [Expected "#0056EC" to be "#0056ec" (color-hex-case)](https://app.codacy.com/gh/WebHeroSchool/-Kostikova-Irina/pullRequest?prid=6356170)